staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -34,6 +37,25 @@ type CustomResourceDefinitionSpec struct {
 	Validation *CustomResourceValidation
 	// Subresources describes the subresources for CustomResources
 	Subresources *CustomResourceSubresources
+	// Versions is the list of all supported versions for this resource.
+	// If Version field is provided, this field is optional.
+	// Validation: All versions must use the same validation schema for now. i.e., top
+	// level Validation field is applied to all of these versions.
+	// Order: The order of these versions is used to determine the order in discovery API
```

Comment:
drop the ordering language from the field doc, since the order here has no bearing on preferred order any more

Reply:
Should we add a reference to APIService.VersionPriority?

Reply:
I'd copy the sorting portion of the comment before adding a reference, but I don't feel super strongly